client-src/elements/form-definition.ts

```diff
@@ -192,6 +195,7 @@ export const FLAT_METADATA_FIELDS: MetadataFields = {
         'name',
         'summary',
         'unlisted',
+        'enterprise_product_category',
```

Comment:
Do you mean for this field to show up on web platform features?  I was thinking that it only applied to features with the enterprise feature_type.

Reply:
ha just removed it, it should only be for enterprises